spec/ruby/language/hash_spec.rb

```diff
@@ -78,14 +78,53 @@
     {rbx: :cool, specs: 'fail_sometimes',}.should == h
   end
 
+  ruby_version_is "2.6" do
```

Comment:
2.6 has released already.

Reply:
sorry, i made a mistake when rebasing.. rebased against latest trunk and updated!